src/pages/FirstKiss/main.ts

```diff
@@ -0,0 +1,107 @@
+import { pageInterface } from '../pageInterface';
+
+const TITLE_LINK_SELECTOR = 'ol.breadcrumb > li:nth-child(3) > a';
+const SELECTED_CHAPTER_SELECTOR =
+  '.header select.single-chapter-select option[selected="selected"]';
+const URL_MANGA_ID_INDEX = 4;
+
+function isChapterPage(url: string): boolean {
+  return url.split('/')[5].startsWith('chapter');
+}
+
+export const FirstKiss: pageInterface = {
+  name: '1stKissManga',
+  domain: 'https://1stkissmanga.me',
+  languages: ['en-US'],
```

Comment:
We are using the name so in this case it would be just "English"

Reply:
Updated